src/app/config.py

```diff
@@ -30,7 +30,8 @@
     raise ValueError(
         "Missing Credentials. Please set 'SUPERUSER_LOGIN' and 'SUPERUSER_PWD' in your environment variables")
 
-BUCKET_NAME: str = "mypyroneartest"
+QARNOT_TOKEN: str = os.getenv("QARNOT_TOKEN")
+BUCKET_NAME: str = "pyronear-media-from-devices-1"
```

Comment:
Just noticed, what is the value supposed to represent? (it's not the bucket name of Qarnot)

Reply:
Yes I am supposing that it is the bucket name of Qarnot; we could change it for sure; did you choose the name of the bucket on Qarnot ? the name has this form (with dashes) because it looks like Qarnot won't accept it otherwise (if I am not wrong)

Reply:
Well yes, it's the one I sent you in the usage instructions :sweat_smile: 
But, I just realized, this cfg param isn't used since the user passes the bucket name as a string to the method/constructor. So can you remove it please?

Reply:
I'm not sure I've understood well, which constructor do you mean ?

Reply:
Respectively:

- my bad, so far it's not used in the constructor since this is an abstract class

- I'm not sure we understand each other: apart from direct access to the bucket, no one should be able to know the bucket information. They are only used so that the API knows where to upload the content. So either we do keep this cfg param using `os.getenv` (**the bucket name or the token should not be exposed on github**), but then it has to be used in your Qarnot bucket service, or we have to pass the cfg value at the top level (most likely in the route). 

My suggestion here:
```
BUCKET_NAME: str =os.getenv('BUCKET_NAME')
```

and in your qarnot bucket service implementation, remove all user inputs regarding the bucket name, and replace them with `cfg.BUCKET_NAME`.

We can set the value securely on Heroku, and for local interaction, we'll pass it as a user input I guess

Reply:
OK I see, thanks !! I hope it is better now